### DISCUSSION TITLE
Residuals after ANCOMBC

### DISCUSSION THREAD

**Discussion by ArtemVorobyev** [FrederickHuangLin/ANCOMBC#177] [Q&A]
Hello,

I would like to ask if it is possible to extract the residuals after the ancombc2 analysis (similar to residuals in linear model)? I would like to use the data for host genome/microbiome interaction analysis. Previously we were using vst data from DESeq2, but can it also be performed with ancombc2?

Regards

Artem

**Comment by FrederickHuangLin** [comment]
Hi Artem,

Unfortunately, `ancombc2` does not export residuals in its current form.

Best,
Huang
